cheungyinglon/hackacola-api
# Feature/user management

Note: meant to be tested with the equivalent pr in: https://github.com/cheungyinglon/hackacola-client

**test instructions:**
1. clone repo
2. `npm install`
3. add `.env.development` to root folder (posted in #website slack channel)
4. run mongo locally and create mongo db (`hackacola-dev` if you'e using the `.env.development` file i shared
5. `npm run dev`

express server should now be running.
